### PR TITLE
(CODEMGMT-117) - modify 1 test, add 1 test.

### PR DESCRIPTION
**Pull request by carlosvelarde**

### CODE REVIEW

File: integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+step 'Push Changes'
+git_add_commit_push(master, 'production', 'Update site.pp and add module.', git_environments_path)
+
+#Tests
+step 'Deploy "production" Environment via r10k with modules'
+on(master, 'r10k deploy environment -p -v')
+
+agents.each do |agent|
+  step "Run Puppet Agent"
+  on(agent, puppet('agent', '--test', '--environment production'), :acceptable_exit_codes => 2)do |result|
```
Comment (cowofevil) [inline integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb:68]: Missing space between ")" and "do".

File: integration/tests/user_scenario/basic_workflow/single_env_upgrade_forge_mod_revert_change.rb [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+#File
+puppet_file_forge = <<-PUPPETFILE
+mod "puppetlabs/motd", '1.1.1'
+PUPPETFILE
+
+puppet_file_git = <<-PUPPETFILE
+mod "puppetlabs/motd",
+  :git => 'https://github.com/puppetlabs/puppetlabs-motd',
+  :tag => '1.2.0'
+PUPPETFILE
```
Comment (cowofevil) [inline integration/tests/user_scenario/basic_workflow/single_env_upgrade_forge_mod_revert_change.rb]: This variable is never used, you should remove it.

File: integration/tests/user_scenario/basic_workflow/single_env_upgrade_forge_mod_revert_change.rb [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+motd_contents_git_regex = /\A#{motd_contents_git}\z/
+
+#File
+puppet_file_forge = <<-PUPPETFILE
+mod "puppetlabs/motd", '1.1.1'
+PUPPETFILE
+
+puppet_file_git = <<-PUPPETFILE
+mod "puppetlabs/motd",
+  :git => 'https://github.com/puppetlabs/puppetlabs-motd',
```
Comment (cowofevil) [inline integration/tests/user_scenario/basic_workflow/single_env_upgrade_forge_mod_revert_change.rb:29]: This test is only about reverting a Forge module. You should have no reference to git at all in this test.

**Comment by cowofevil** [comment]
@carlosvelarde  The 'CODEMGMT-117 - C63602- Single Environment Upgrade Forge Module then Revert Change' test has fundamental problems. The idea behind the test is that you start with an old pinned version of the module, upgrade to the latest version and then revert back to the original pinned version. Currently you're using git reference in the module and it needs to be changed to use the forge instead.

File: integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    assert_no_match(/Error:/, result.stderr, 'Unexpected error was detected!')
+  end
```
Comment (cowofevil) [inline integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb:70]: You fail to verify the contents of the MOTD file after the puppet run.

**Comment by carlosvelarde** [comment]
Closed in favor of 364.

**Pull request closed without merging**